projects/ebv-analyzer-app/src/app/ebv-selector/ebv-selector.component.ts

```diff
@@ -209,6 +213,92 @@ export class EbvSelectorComponent implements OnInit, OnDestroy {
+        ])
+            .pipe(
+                first(),
+                tap(() => {
+                    this.plotLoading.next(true);
+                    this.plotData.next(undefined);
+                }),
+                mergeMap(([rasterWorkflow, selectedCountry]) =>
+                    this.projectService.registerWorkflow({
+                        type: 'Plot',
+                        operator: {
+                            type: 'MeanRasterPixelValuesOverTime',
+                            params: {
+                                timePosition: 'start',
+                                area: false,
+                            },
+                            sources: {
+                                raster: {
+                                    type: 'Expression',
+                                    params: {
+                                        expression: 'if B IS NODATA { out_nodata } else { A }',
+                                        outputType: RasterDataTypes.Float64.getCode(),
+                                        outputNoDataValue: 'nan',
+                                        mapNoData: false,
+                                    },
+                                    sources: {
+                                        a: rasterWorkflow.operator,
+                                        b: {
+                                            type: 'GdalSource',
+                                            params: {
+                                                dataset: COUNTRY_DATA_LIST[selectedCountry.name].raster,
+                                            },
+                                        },
+                                    },
+                                } as ExpressionDict,
+                            },
+                        } as MeanRasterPixelValuesOverTimeDict,
+                    }),
+                ),
+                mergeMap((workflowId) =>
+                    combineLatest([
+                        of(workflowId),
+                        this.userService.getSessionTokenForRequest(),
+                        this.projectService.getSpatialReferenceStream(),
+                        this.mapService.getViewportSizeStream(),
+                    ]),
+                ),
+                mergeMap(([workflowId, sessionToken, crs, viewport]) =>
+                    this.backend.getPlot(
+                        workflowId,
+                        {
+                            time: {
+                                start: (this.ebvTree as EbvHierarchy).tree.time.start,
+                                end: (this.ebvTree as EbvHierarchy).tree.time.end,
+                            },
+                            bbox: extentToBboxDict(viewport.extent),
+                            crs: crs.srsString,
+                            // TODO: set reasonable size
+                            spatialResolution: [0.1, 0.1],
```

Comment:
Do we need to set this based on our dataset? Or on our bounds?

Reply:
I am not sure. The plot isn't responsive to the map so we should just use some reasonable numbers here

Reply:
I guess we could do it in the original resolution of the dataset. I'm not sure if it is slow then.